internal/cloudsql/instance.go

```diff
@@ -225,6 +234,16 @@ func (c ConnectionInfo) Addr(ipType string) (string, error) {
 // TLSConfig constructs a TLS configuration for the given connection info.
 func (c ConnectionInfo) TLSConfig() *tls.Config {
 	pool := x509.NewCertPool()
+	if c.ServerCaMode == "GOOGLE_MANAGED_CAS_CA" {
+		pool.AppendCertsFromPEM(c.ServerCaCertPem)
+		return &tls.Config{
+			ServerName:         c.DNSName,
+			Certificates:       []tls.Certificate{c.ClientCertificate},
+			RootCAs:            pool,
+			InsecureSkipVerify: false,
```

Comment:
This line can be omitted and we can just used the zero value.

Reply:
SG. Just removed it. Thanks.